### PR TITLE
Information complement

### PR DESCRIPTION
**Pull request by q384566678**
Some information complements

Signed-off-by: zhouhao zhouhao@cn.fujitsu.com

### CODE REVIEW

**Comment by wking** [comment]
52db7f57 looks good to me.

**Comment by Mashimiao** [comment]
ping @opencontainers/image-tools-maintainers

File: cmd/oci-create-runtime-bundle/oci-create-runtime-bundle.1.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ oci-create-runtime-bundle \- Create an OCI runtime bundle
 
 # SYNOPSIS
 **oci-create-runtime-bundle** [src] [dest] [flags]
+**oci-create-runtime-bundle** [--help/--version]
```
Comment (Mashimiao) [inline cmd/oci-create-runtime-bundle/oci-create-runtime-bundle.1.md]: [--help|--version]  will be better

Comment (q384566678) [inline cmd/oci-create-runtime-bundle/oci-create-runtime-bundle.1.md]: updated

**Comment by caniszczyk** [comment]
LGTM

**Comment by stevvooe** [comment]
LGTM

**Pull request merged**